src/ui/map.ts

```
@@ -3381,7 +3383,11 @@ export class Map extends Camera {
                 PerformanceUtils.frame(paintStartTimeStamp);
                 this._frameRequest = null;
                 this._render(paintStartTimeStamp);
-            }).catch(() => {}); // ignore abort error
+            }).catch((error: Error) => {
+                if (!isAbortError(error) && !isFramebufferNotCompleteError(error)) {
```

Don't like swallowing "framebuffer is not complete" errors but I haven't been able to make some tests work doing it any other way. Mocking `WebGLContext`, which would be the best solution, doesn't seem to work on some instances.
Although we swallow that error, I think this is an improvement over not reporting any of them